sidebars.js

```diff
@@ -12,59 +11,65 @@
 
 module.exports = {
   docs: [
-    'introduction',
+    "introduction",
     {
-      type: 'category',
-      label: 'Getting started',
+      type: "category",
+      label: "Getting started",
       collapsed: false,
       items: [
-        'community/welcome-to-the-community',
-        'community/highlights',
-        'community/100-days-of-oss',
-        'community/faqs'
+        "community/welcome-to-the-community",
+        "community/highlights",
+        "community/100-days-of-oss",
+        "community/faqs",
       ],
     },
     {
-      type: 'category',
-      label: 'Contributing guide',
+      type: "category",
+      label: "Contributing guide",
       collapsed: false,
       items: [
-        'contributing/introduction-to-contributing',
-        'contributing/code-of-conduct',
-        'contributing/triage-guide',
-        'contributing/set-up-authentication'
+        "contributing/introduction-to-contributing",
+        "contributing/code-of-conduct",
+        "contributing/triage-guide",
+        "contributing/set-up-authentication",
       ],
     },
     {
-      type: 'category',
-      label: 'Technical guide',
+      type: "category",
+      label: "Technical guide",
       collapsed: false,
-      items: [
-        'technical/introduction-to-storybook',
-        'technical/resolve-merge-conflicts',
-      ],
+      items: ["technical/introduction-to-storybook", "technical/resolve-merge-conflicts"],
     },
     {
-      type: 'category',
+      type: "category",
       label: `Maintainer guide`,
       collapsed: true,
       items: [
-        'maintainers/setting-up-a-new-repository',
-        'maintainers/check-engines',
-        'maintainers/conventional-commit',
-        'maintainers/semantic-release',
-      ]
+        "maintainers/setting-up-a-new-repository",
+        "maintainers/check-engines",
+        "maintainers/conventional-commit",
+        "maintainers/semantic-release",
+      ],
     },
     {
-      type: 'category',
-      label: 'Chrome Extension Usage',
+      type: "category",
+      label: "Chrome Extension Usage",
       collapsed: true,
       items: [
-        'chrome-extension/introduction-to-the-chrome-extension',
-        'chrome-extension/using-the-chrome-extension',
+        "chrome-extension/introduction-to-the-chrome-extension",
+        "chrome-extension/using-the-chrome-extension",
+        {
+          type: "folder",
+          label: "using the extension",
+          items: [
+            {
+              type: "item",
+              label: "feature",
+              collapsed: ,
```

Comment:
@CBID2 this is causing a build failure. Collapsed needs a value.

Reply:
Ahh ok. So I'm on the right track then @bdougie?

Reply:
Yeah, looks pretty well structured to me. We do have another error though - https://app.netlify.com/sites/docs-open/deploys/64c04a7f16acc50008883d18

Reply:
Are you sure `folder` is a correct type here - https://github.com/open-sauced/docs/pull/123/files#diff-9a6a0fdafefb0eecffb77784bc94f5ad984062167b8bcb7c4a899c90f6f567e8R62? I checked the docs and maybe even that should be `category` https://docusaurus.io/docs/sidebar?

Reply:
I can try that. Thanks @diivi